api/models/Survey.js

```diff
@@ -17,6 +17,8 @@ module.exports = class Survey extends Model {
                         models.Survey.hasMany(models.SurveyResult);
                         models.Survey.belongsToMany(models.Region, {through: 'surveyregion'});
                         models.Survey.belongsToMany(models.Office, {through: 'surveyoffice'});
+                        models.QuestionType.hasOne(models.Survey);
```

Comment:
do we support mixed question types? would be support having many question types?